### PR TITLE
Prepare for v0.5.0 release

### PR DESCRIPTION
**Pull request by jazz-cb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
+### [0.4.0] - 2024-09-11
```
Comment (alex-stone) [inline CHANGELOG.md:10]:
```suggestion
### [0.5.0] - 2024-09-11
```

Comment (alex-stone) [reply CHANGELOG.md:10]: Might want to add a line for `### Skipped 0.4.0` somewhere here too?

Comment (jazz-cb) [reply CHANGELOG.md:10]: sure

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
+### [0.5.0] - 2024-09-11
```
Comment (alex-stone) [inline CHANGELOG.md:10]:
```suggestion
## [0.5.0] - 2024-09-11
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-- Add support for Arbitrum-Mainnet
+- Add Arbitrum-Mainnet support for Native transfers.
 - Add optional arguments to allow setting amount for payable contract method invocations
 
+### [0.4.0] - Skipped
```
Comment (alex-stone) [inline CHANGELOG.md:15]:
```suggestion
## [0.4.0] - Skipped
```

**Review by rohan-agarwal-coinbase — APPROVED** [review]

**Pull request merged**